README.md

```diff
@@ -6,25 +6,25 @@
 [![Support](https://img.shields.io/badge/$-support-12a0df.svg?style=flat)](https://aenix.io/contact-us/#meet)
 [![Active](http://img.shields.io/badge/Status-Active-green.svg)](https://aenix.io/cozystack/)
 [![GitHub Release](https://img.shields.io/github/release/aenix-io/cozystack.svg?style=flat)](https://github.com/aenix-io/cozystack)
-[![GitHub Commit](https://img.shields.io/github/commit-activity/y/aenix-io/cozystack)](https://github.com/aenix-io/cozystack) 
+[![GitHub Commit](https://img.shields.io/github/commit-activity/y/aenix-io/cozystack)](https://github.com/aenix-io/cozystack)
 
 # Cozystack
 
 **Cozystack** is a free PaaS platform and framework for building clouds.
 
 With Cozystack, you can transform your bunch of servers into an intelligent system with a simple REST API for spawning Kubernetes clusters, Database-as-a-Service, virtual machines, load balancers, HTTP caching services, and other services with ease.
 
-You can use Cozystack to build your own cloud or to provide a cost-effective development environments.  
+You can use Cozystack to build your own cloud or to provide a cost-effective development environments.
 
 ## Use-Cases
 
-* [**Using Cozystack to build public cloud**](https://cozystack.io/docs/use-cases/public-cloud/)  
+* [**Using Cozystack to build public cloud**](https://cozystack.io/docs/use-cases/public-cloud/)
 You can use Cozystack as backend for a public cloud
 
-* [**Using Cozystack to build private cloud**](https://cozystack.io/docs/use-cases/private-cloud/)  
+* [**Using Cozystack to build private cloud**](https://cozystack.io/docs/use-cases/private-cloud/)
 You can use Cozystack as platform to build a private cloud powered by Infrastructure-as-Code approach
 
-* [**Using Cozystack as Kubernetes distribution**](https://cozystack.io/docs/use-cases/kubernetes-distribution/)  
+* [**Using Cozystack as Kubernetes distribution**](https://cozystack.io/docs/use-cases/kubernetes-distribution/)
 You can use Cozystack as Kubernetes distribution for Bare Metal
```

Comment:
Markdown syntax is very tricky, these two examples are rendered differently:

with `  ` in the end:
![Screenshot 2024-10-11 at 9 26 04](https://github.com/user-attachments/assets/c895dbfe-c80f-43a5-9496-6b241cb4c08e)

without `  ` in the end it sticks lines together:

![Screenshot 2024-10-11 at 9 25 50](https://github.com/user-attachments/assets/25cde49f-78fd-4c43-b882-7a79acbde7e6)